qupath.github.io.gemspec

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+    spec.name          = "qupath.github.io"
+    spec.version       = "0.1.0"
+    spec.authors       = ["finglis"]
+    spec.email         = ["42358257+finglis@users.noreply.github.com"]
+  
+    spec.summary       = "TODO: Write a short summary, because Rubygems requires one."
+    spec.homepage      = "TODO: Put your gem's website or public repo URL here."
+    spec.license       = "MIT"
+  
+    spec.files         = `git ls-files -z`.split("\x0").select { |f| f.match(%r!^(assets|_layouts|_includes|_sass|LICENSE|README)!i) }
+  
+    spec.add_runtime_dependency "jekyll", "~> 3.8"
+  
+    spec.add_development_dependency "bundler", "~> 1.16"
+    spec.add_development_dependency "rake", "~> 12.0"
+  end
```

Comment:
Why do we need a gemspec?

Reply:
Yeah, this is for publishing to rubygems/installing the repo as a rubygem, sort of like an R package DESCRIPTION or a python pyproject.toml. bundler is handling development and runtime dependencies for us so we don't need it here